micropython_ota/micropython_ota.py

```diff
@@ -0,0 +1,45 @@
+import machine
+from urequests import urequests
+import os
+
+
+def check_version(host, project):
+    try:
+        if 'version' in os.listdir():
+            with open('version', 'r') as current_version_file:
+                current_version = current_version_file.readline().strip()
+        else:
+            current_version = ''
+        remote_version_response = urequests.get(f'{host}/{project}_version')
```

Comment:
Instead of prefixing the version file with the project name, using a context path would be more suitable.